.mailmap

```diff
@@ -0,0 +1,2804 @@
+Aakash Jain <aakash_jain@apple.com> <aakash_jain@apple.com>
```

Comment:
I realise `validate-committer-lists`produces this output, but this is equivalent to 
```suggestion
Aakash Jain <aakash_jain@apple.com>
```

(i.e., change the name of any author/commit with the email `aakash_jain@apple.com` to `Aakash Jain`)